lib/runner/Runner.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "ParamController.hpp"
+#include "interface/InterfaceController.hpp"
+
+class Runner {
+ public:
+  Runner(Poles poles, ParamController* param_controller);
```

Comment:
I would prefer to make this accept and store a reference to the `ParamController` instead of a pointer. This makes it clear that the parameter cannot be null.

Reply:
Done.